src/resources/views/crud/form_content.blade.php

```diff
@@ -80,10 +80,19 @@ function triggerFocusOnFirstInputField(firstField) {
      * @param {jQuery} firstField
      */
     function handleFocusOnSelect2Field(firstField){
-        $('.select2-search__field').remove();
         firstField.select2('focus');
     }
 
+    /*
+    * Hacky fix for a bug in select2 with jQuery 3.6.0's new nested-focus "protection"
+    * see: https://github.com/select2/select2/issues/5993
+    * see: https://github.com/jquery/jquery/issues/4382
+    *
+    */
+    $(document).on('select2:open', () => {
+        setTimeout(() => document.querySelector('.select2-container--open .select2-search__field').focus(), 200);
```

Comment:
I am always a bit concerned when we use `setTimeout` with a specific time and no fallback. 
If it takes 201ms to update the DOM (maybe dev has some heavy calculations or something before) it will fail and it will be very very difficult to debug.

Usually what I think it's better in this case is using recursion, we define the setTimeout as a constant, and in the setTimeout body we check if `document.querySelector('.select2-container--open .select2-search__field')` exists already. (we do it every 50/100ms or so)

Probably `setInterval` is the best approach here? We can run every 50ms and then call `clearInterval(intervalId)` to stop when we found the element and focused, or after 1000ms if not found we can log a developer error so we know why something is not working related with this functionality ?

Let me know what you think about this. I think it would help us to prevent future difficult to debug issues. 

Cheers

Reply:
Hmm... that sounds safer but... is that really needed, I wonder? This isn't rocket science, nor a business critical operation. Don't know, you guys decide, just be careful not to over-engineer this. My 2 cents.

Reply:
Agreed, I am waiting on @jorgetwgroup to give is opinion too. 

This is not critical until the moment: `my select2 sometimes open, sometimes don't` -> 199ms vs 201ms.

Maybe 200ms is too much to hang the user operation  ? Because sometimes 50ms should be more than enough to get ready to go ? 

I have no problem to let this go this way, we already did it before ... I just wanted to let my concerns clear, maybe @jorgetwgroup have a "not so overenginereed" solution for this. 👍 

Cheers

Reply:
To be honest, i see select2 library and didnt receive any update from 2020, so overthinking this small fix while make us loose our time, btw the big change is delete the "remove" line, because thats make multiple didnt work at all.

So as "Gold & Silver Pawn Shop" we will be in the middle, i offer 200, you offer 50, let it on 100 ms i push commit.

Cheers.